examples build issue on linux

As library it seems to compile so there might be some issue in a templated function calling some now removed function maybe.

```
/tmp/.dub/build/dopt-0.3.6+commit.2.gae525f7/library-debug-linux.posix-x86_64-dmd_2078-17F6D836CB67D7791574F1469CA9ED39/libdopt.a(math_2bd_4f1.o): In function `_D4dopt4core3cpu4math12matmulKernelFCQBjQBh3ops9OperationAxSQCgQCe5types6BufferSQDaQCyQuQqZv':
/home/webfreak/Development/dopt/examples/../source/dopt/core/cpu/math.d:66: undefined reference to `cblas_sgemm'
collect2: error: ld returned 1 exit status
Error: linker exited with status 1
/usr/bin/dmd failed with exit code 1.
```

Got blas & cblas from archlinux `extra` repository (3.8.0, part of lapack)

--- a/dub.json
+++ b/dub.json
@@ -13,7 +13,7 @@
 		"derelict-cudnn": "~>2.0.1"
 	},
 	"libs": [
-		"blas"
+		"blas", "cblas"
 	],
 	"-ddoxTool": "scod",
 	"-ddoxFilterArgs": [

